origami.json

```
@@ -3,7 +3,7 @@
 	"origamiType": "component",
 	"supportStatus": "active",
 	"brands" : [
-		"master"
+		"main"
```

This isn't a branch name unfortunately, it is a brand name -- the brand name "master" is being renamed at some point but that work hasn't been finished just yet

```suggestion
		"master"
```

---

👍 [Here's the proposal](https://github.com/Financial-Times/origami/pull/65). I think the Brand team decided on renaming the master brand to core, I'll make a note to follow up with them to see if they have gone ahead with that

---

so renaming this will fall under your migration work in future?

---

Yes 👍